drivers/timer/xlnx_psttc_timer_priv.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019 Stephanos Ioannidis <root@stephanos.io>
+ * Copyright (c) 2018 Xilinx, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_TIMER_XLNX_PSTTC_TIMER_PRIV_H_
+#define ZEPHYR_DRIVERS_TIMER_XLNX_PSTTC_TIMER_PRIV_H_
+
+/*
+ * Triple-timer Counter (TTC) Register Offsets
```

Comment:
Dumb question: is there a link to a public reference for this hardware?  This would be a good place to put that if you have it.

Reply:
The Xilinx TTC (well, it is a Cadence IP core that has been customised by Xilinx) is used on the Xilinx ZynqMP (now called UltraScale+) SoC.

"Zynq UltraScale+ Device Technical Reference Manual" provides documentation for the TTC.